components/swatch/index.css

```diff
@@ -10,38 +10,87 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-.🤫 {
-  /* hack: make sure we don't complain about this color being undefined */
-  --spectrum-picked-color: 0;
+/* Swatch tokens */
+.spectrum-Swatch {
+  /* Static properties - no tokens */
+  --spectrum-swatch-focus-ring-border-radius: 8px;
+
+  --spectrum-swatch-border-color-regular: rgba(0, 0, 0, 0.5);
```

Comment:
I don't think the focus ring border radius is missing. I think it could be done either as

`--spectrum-swatch-focus-ring-border-radius: calc(var(--spectrum-corner-radius-100) * 2);`
or 
`--spectrum-swatch-focus-ring-border-radius: var(--spectrum-corner-radius-200);`

Both would result in 8px;